Blockzilla/AppDelegate.swift

```diff
@@ -18,6 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             BuddyBuildSDK.setup()
         #endif
 
+        // Set up Telemetry
+        let telemetryConfig = Telemetry.default.configuration
+        telemetryConfig.appName = "Focus"
```

Comment:
Is this a value that we send to the telemetry endpoint? If so then we probably want to send *Focus* vs *Klar* depending on the build. (See `AppConfig.productName`)